apps/backend/api/models/Group.js

```diff
@@ -741,7 +993,8 @@ module.exports = bookshelf.Model.extend(merge({
 
       await group.createStarterPosts(trx)
 
-      await group.createInitialWidgets(trx)
+      // await group.createInitialWidgets(trx)
```

Comment:
Note: haven't cut out the prior 'widget' stuff. Until everything has transitioned, we'll still need it